### DISCUSSION TITLE
Script Mappers and Authentication Flows in KC 17.0.0

### DISCUSSION THREAD

**Discussion by td-trt** [keycloak/keycloak#10812] [Archive]
Hello,

I am trying to upgrade from 16.1.1 to 17.0.0 and part of that is porting across our script mappers and authentication flow executions. 

I have followed the instructions here: https://www.keycloak.org/docs/latest/server_development/#create-a-jar-with-the-scripts-to-deploy 

I have also tried setting the scripts and upload-scripts to enabled using:

`kc.sh build --features=scripts,upload-scripts`

KC 17.0.0 then builds and starts as expected. The problem is that the mappers (under clients) and the executions under Authentication Flows simply don't show up. 

No errors in the log file so no pointers to where the problem is.

What have I missed? Any ideas?

Really hope somebody has solved this one.

Thanks in advance.

TD

**Comment by axeh** [comment]
I am using Keycloak 19.0.3 and I have the same problem. Nothing in the logs t
